modules/configuring-disk-sharing-by-using-LUN.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-disk-sharing-by-using-LUN_{context}"]
+= Configuring disk sharing by using LUN
+
+You can configure a LUN-backed block mode virtual machine disk to be shared among multiple virtual machines by enabling SCSI persistent reservation. Enabling this option allows you to use advanced SCSI commands, such as those required for a Windows Failover Clustering implementation, against the underlying storage.
+
+A disk of type `LUN` exposes the volume as a LUN device to the VM. This allows the VM to execute arbitrary iSCSI command passthrough on the disk. 
+
+You reserve a LUN through the SCSI Persistent Reserve options to protect data on the VM from outside access. In order to issue privileged, SCSI device-specific input and output controls (IOCTLs), the VM requires activation of the `PersistentReservation` option on the LUN disk. You then enable the reservation by configuring the `featureGates` option. 
```

Comment:
nit: if you want to reference what are the scis commands, you can use a reference to the QEMU documentation: https://www.qemu.org/docs/master/interop/pr-helper.html